### PR TITLE
add evaluation admin progress screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Admin Progress Screen#187](https://github.com/nerubia/kh360-react/issues/187)

BE PR: [update get evaluations and send reminder apis #191](https://github.com/nerubia/kh360-nodejs/pull/191)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/[id]/evaluees/[evaluation_result_id]/evaluators/EvaluatorsList.tsx
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/evaluees/[evaluation_result_id]/evaluators/EvaluatorsList.tsx:1]: update the file name

File: src/features/admin/evaluation-administrations/[id]/evaluees/[evaluation_result_id]/evaluators/EvaluatorsRoles.tsx
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/evaluees/[evaluation_result_id]/evaluators/EvaluatorsRoles.tsx:1]: update the file name

File: src/features/admin/evaluation-administrations/[id]/preview/PreviewEmployeesTable.tsx
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/preview/PreviewEmployeesTable.tsx:1]: update the file name

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      if (firstNameA !== firstNameB) {
+        return firstNameA.localeCompare(firstNameB)
+      }
+
+      return lastNameA.localeCompare(lastNameB)
+    })
+    setSortedEvaluators(sorted)
+  }, [evaluators])
+
+  const [evaluatorToggledState, setEvaluatorToggledState] = useState<boolean[]>([])
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:49]: Move this above the useEffect

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        void appDispatch(
+          getEvaluations({ evaluation_administration_id: id, evaluator_id, for_evaluation: true })
+        )
+      }
+      setDispatchedEmployees((prevDispatchedEmployees) => [...prevDispatchedEmployees, index])
+      setSelectedEvaluatorId(evaluator_id)
+    }
+  }
+
+  useEffect(() => {
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:66]: move this too

File: src/types/evaluation-administration-type.ts [hunk 3]
```diff
@@ -28,6 +28,11 @@ export interface ExternalEvaluatorData {
   external_user_ids: number[]
 }
 
+export interface sendReminderData {
```
Comment (jatnerubia) [inline src/types/evaluation-administration-type.ts:31]: use pascal here

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/redux/slices/usersSlice.ts
Comment (jatnerubia) [inline src/redux/slices/usersSlice.ts:1]: refactor this

File: src/redux/slices/authSlice.ts
Comment (jatnerubia) [inline src/redux/slices/authSlice.ts:1]: refactor this

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**